x-pack/plugins/lens/public/pie_visualization/render_function.tsx

```diff
@@ -198,8 +198,6 @@ export function PieComponent(
     setState({ isReady: true });
   }, []);
 
-  const reverseGroups = [...columnGroups].reverse();
```

Comment:
It seems like the logic of "groups" can be simplified further (line 75 of this file) because there can't be any sub-totals in between the known columns in the data